modules/account-lib/src/coin/stx/transactionBuilder.ts

```diff
@@ -0,0 +1,251 @@
+import BigNumber from 'bignumber.js';
+import BigNum from 'bn.js';
+import { BaseCoin as CoinConfig } from '@bitgo/statics/dist/src/base';
```

Comment:
```suggestion
import { BaseCoin as CoinConfig } from '@bitgo/statics';
```

I think this is exported from the top level.